Fix: Add 'back' button instead of 'cancel' to session request page

meadmaker opened the pull request:
The 'Cancel' button that goes back to the same page was confusing, so change it out for a 'Back' button instead.

Fixes #4007

meadmaker committed:
Fix: Add 'back' button instead of 'cancel' to session request page

The 'Cancel' button that goes back to the same page was confusing, so change it out for a 'Back' button instead.

Fixes #4007

larseggert commented on ietf/secr/templates/includes/sessions_request_form.html:
```diff
@@ -118,7 +118,7 @@
         <ul>
             <li><button type="submit" name="submit" value="Save"{% if is_locked %} disabled{% endif %}>Save</button></li>
-            <li><button type="submit" name="submit" value="Cancel">Cancel</button></li>
+            <li><button type="button" onclick="window.location='{% url "ietf.secr.sreq.views.main" %}'">Back</button></li>
```
This should simply be an `a` tag with an `href`, rather than needing js.

larseggert commented on ietf/secr/templates/includes/sessions_request_form.html:
```diff
@@ -118,7 +118,7 @@
         <ul>
             <li><button type="submit" name="submit" value="Save"{% if is_locked %} disabled{% endif %}>Save</button></li>
-            <li><button type="submit" name="submit" value="Cancel">Cancel</button></li>
+            <li><button type="button" onclick="window.location='{% url "ietf.secr.sreq.views.main" %}'">Back</button></li>
```
```suggestion
            <li><a type="button" href="{% url 'ietf.secr.sreq.views.main' %}">Back</a></li>
```

larseggert replied:
Or does this template not use bs5?

meadmaker replied:
Bootstrap is available, but not used on the overall template.  I could pull it in if desired.

If so, then I'll probably make the same changes to `ietf/secr/templates/sreq/view.html` as well, since I copied it out of there.

meadmaker replied:
Oh, I just realized this is wrong anyway - I should have it go back through the history instead of specifying a specific target, because this form is included on two paths, which have different navigation.

meadmaker committed:
Fix: Change absolute target for going back with history.back()

This form can be included from multiple pages, which have differing
navigation for how to land on the form.  Therefore, history.back()
is the single way to navigate back to the previous page.

meadmaker committed:
Merge branch 'ietf-tools:main' into mark/new_session_back_button

meadmaker commented:
I've now made that change, and it's ready for re-review.

rjsparks commented:
Do not put the effort into bs5 this template. We will be replacing /secr/sreq with a reimplementation in /meeting later instead.
If this has become complicated to address, just remove the Cancel button and leave it to the user to use their browser back button.
